src/types/form.ts

```diff
@@ -264,7 +264,7 @@ export type UseFormResetField<TFieldValues extends FieldValues> = <
 ) => void;
 
 export type UseFormReset<TFieldValues extends FieldValues> = (
-  values?: DefaultValues<TFieldValues>,
+  values?: DefaultValues<TFieldValues> | UnpackNestedValue<TFieldValues>,
```

Comment:
I suggest the following fixes.
Does this fix not solve the problem?

```suggestion
  values?: UnpackNestedValue<DefaultValues<TFieldValues>>,
```

Reply:
![Screen Shot 2021-12-17 at 12 59 06 pm](https://user-images.githubusercontent.com/10513364/146476360-c022a6ce-bbb6-40a6-ae7f-b316774c014c.png)

causing a type error ^